forte/data/ontology/configs/example_import_ontology_config.json

```diff
@@ -1,27 +1,31 @@
 {
-    "name": "ft.onto.example_import_ontology",
-    "definitions": {
-        "Token": {
+    "name": "example_import_ontology",
+    "definitions": [
+        {
             "namespace": "ft.onto.example_import_ontology.Token",
```

Comment:
- This design can not achieve what we planned. Our decision is to add a prefix to all user-generated types (ft.onto) by default. But if we are asking the user to write ``ft.onto``every time, this will not be enforced.  The ``ft.onto`` should be inserted automatically by default. If the user wants to change it, he can use a top-level non-empty field ``prefix``, to change that (so the default behavior is "prefix": "ft.onto")

- This is not a namespace, this is simply a name. Please name it ``entry_name``. See https://github.com/hunterhector/forte/blob/master/forte/data/ontology/configs/stanfordnlp_ontology.json